models/bases/appointment_schedules.sql

```diff
@@ -0,0 +1,12 @@
+-- May include appointment schedules for the test patient.
+select
+    s.id,
+    s.until_date,
+    s.interval,
+    s.days_of_week,
+    s.frequency,
+    s.nth_weekday,
+    s.occurrence_count,
+    s.is_fully_generated
+from {{ source("tamanu", "appointment_schedules") }} s
+where s.deleted_at is null
```

Comment:
Exclude test patient appointment schedules similar to what we do for lab tests.
```suggestion
from {{ source("tamanu", "appointment_schedules") }} s
join {{ source("tamanu", "appointments") }} a on a.schedule_id = s.id
where s.deleted_at is null
    and a.patient_id != '{{ var("test_patient") }}'
```

Reply:
Ahh, it's not possible to directly extract test patients from the appointment_schedules table by joining it with the appointments model. The reason is that the appointments model contains one record for each occurrence of an appointment. If a schedule is set to repeat three times, the appointments table will have three records—one for each occurrence—resulting in multiple records in this table. That is why I opted to skip it for now and added a note at top 😬

Given the situation, we have two options:
- Still have a join and do DISTINCT on s.id in the SELECT statement
- Have a subquery to get DISTINCT schedule ids for non-test patients appointments and use it for WHERE clause in main SELECT statement.

Which approach do think would be right? @julianam-w

Reply:
As discussed we've gone with merging `appointment_schedules` into `appointments`. 👍